salvo/src/lib/test_source_manager.py

```diff
@@ -40,23 +77,57 @@ def test_get_envoy_images_for_benchmark(mock_run_command):
   """Verify that we can determine the current and previous image
```

Comment:
(pre-existing) Should this method be called `test_get_envoy_hashes_for_benchmark` since that is the name of the method it is testing?

Reply:
I'll update this